client/src/lib/validationComplete.js

```diff
@@ -0,0 +1,11 @@
+export const validationComplete = inputs => {
+  let valid = true;
+
+  Object.keys(inputs).forEach(key => {
+    if (inputs[key] === false) {
+      valid = false;
+    }
+  });
+
+  return valid;
+};
```

Comment:
Does this apply for all forms? You probably want to have a custom validation function for every form.

On that note, there doesn't seem to be much of a reason why this needs to be passed as a prop. If you're going to use a single validation functions for all fields, why not just import it directly into the component file?

Reply:
:+1: for moving the function into the component. 

what would the advantage be to a separate function for each form?

Reply:
I guess if something lives in the `inputValidation` object, then if any of them are false, this function should return false, so this would be fine in that case...

You might find certain situations where one field is invalid but it's okay because another field is not (e.g. where you need to fill in one OR the other). You could potentially create a form-specific validation function in that case, though.

Reply:
BTW, you can rewrite this file like:
```
export const validationComplete = inputs => Object.values(inputs).every(Boolean)
```
This starts by creating an array of the values (since the key names don't matter) and determining whether all of them are truthy or not.